mpas_analysis/configuration/MpasAnalysisConfigParser.py

```diff
@@ -66,29 +64,24 @@ def getExpression(self, section, option, elementType=None,
         of having selected numpy and / or np functionality available.
 
         Author: Xylar Asay-Davis, Phillip J. Wolfram
-        Last Modified: 04/10/2017
         """
-        if self.has_section(section):
-            if self.has_option(section, option):
-                expressionString = self.get(section, option)
-                if usenumpyfunc:
-                    assert '__' not in expressionString, \
-                            "'__' is not allowed in {} "\
-                            "for `usenumpyfunc=True`".format(expressionString)
-                    sanitizedstr = expressionString.replace('np.', '')\
-                                                   .replace('numpy.', '')\
-                                                   .replace('__', '')
-                    result = eval(sanitizedstr, npallow)
-                else:
-                    result = ast.literal_eval(expressionString)
+        expressionString = self.get(section, option)
+        if usenumpyfunc:
+            assert '__' not in expressionString, \
+                    "'__' is not allowed in {} "\
+                    "for `usenumpyfunc=True`".format(expressionString)
+            sanitizedstr = expressionString.replace('np.', '')\
+                                           .replace('numpy.', '')\
+                                           .replace('__', '')
+            result = eval(sanitizedstr, npallow)
+        else:
+            result = ast.literal_eval(expressionString)
 
-                if elementType is not None:
-                    if isinstance(result, (list, tuple)):
-                        result = [elementType(element) for element in result]
-                    elif isinstance(result, dict):
-                        for key in result:
-                            result[key] = elementType(result[key])
+        if elementType is not None:
+            if isinstance(result, (list, tuple)):
+                result = [elementType(element) for element in result]
+            elif isinstance(result, dict):
+                for key in result:
+                    result[key] = elementType(result[key])
 
-                return result
-        else:
-            return None
```

Comment:
Instead of returning `None` we report an error in the call to `self.get(section, option)` if there is no `option` for the requested `section`, correct?  This follows from our discussion earlier this week, right?